workflow/scripts/wrapper.py

```diff
@@ -0,0 +1,49 @@
+#!/usr/bin/env python3
+
+import pandas as pd
+import glob
+import shutil
+import os
+import yaml
+
+with open('config/wrapper_config.yaml', 'r') as f:
```

Comment:
Id' also put all of the below into a function to better handle any weird business that might arise.